buildspec/windowsTests.yml

```diff
@@ -16,6 +16,24 @@ phases:
   post_build:
     commands:
       - |
+        $env:TEST_ARTIFACTS=Join-Path $env:TEMP testArtifacts
```

Comment:
this could be script-scoped (`$script:TEST_ARTIFACTS`) instead of an env var, unless the intention is to make TEST_ARTIFACTS available in the environment of child processes.

Reply:
Changed